api/src/pcapi/routes/native/v1/feature_toggles.py

```diff
@@ -0,0 +1,24 @@
+import logging
+
+from pcapi import settings
+from pcapi.models.api_errors import ApiErrors
+from pcapi.models.feature import Feature
+from pcapi.serialization.decorator import spectree_serialize
+
+from . import blueprint
+from .serialization import features_toggle as serializers
+
+
+logger = logging.getLogger(__name__)
+
+
+@blueprint.native_v1.route("/features_toggle", methods=["PATCH"])
```

Comment:
(détail / question de goût) Pas fan du chemin. Puisqu'on fait un PATCH, on pourrait simplifier le chemin : 

```suggestion
@blueprint.native_v1.route("/features", methods=["PATCH"])
```